File: problem/2d_streamer/codegen_test/codegen/mex/mathOpsIntegrated/mathOpsIntegrated.c
/*
 * Academic License - for use in teaching, academic research, and meeting
 * course requirements at degree granting institutions only.  Not for
 * government, commercial, or other organizational use.
 *
 * mathOpsIntegrated.c
 *
 * Code generation for function 'mathOpsIntegrated'
 *
 */

/* Include files */
#include "mathOpsIntegrated.h"
#include "rt_nonfinite.h"
#include "flux.h"

/* Function Definitions */
void mathOpsIntegrated(const emlrtStack *sp, real_T in1, real_T in2,
                       real_T *added, real_T *multed)
{
  (void)sp;
  /*  for code generation, preinitialize the output variable */
  /*  data type, size, and complexity  */
  /*  generate an include in the C code */
  /*  evaluate the C function */
  *added = flux(in1, in2);
  *multed = in1 * in2;
}

/* End of code generation (mathOpsIntegrated.c) */

File: problem/2d_streamer/codegen_test/mathOpsIntegrated.m
function [added, multed] = mathOpsIntegrated(in1, in2)
    %#codegen
    % for code generation, preinitialize the output variable
    % data type, size, and complexity 
    added = 0;
    % generate an include in the C code
    coder.cinclude('flux.h');
    % evaluate the C function
    added = coder.ceval('flux', in1, in2); 
    multed = in1*in2; 
    end

    